zyte_spider_templates_project/settings.py

```diff
@@ -6,28 +6,28 @@
 BOT_NAME = "zyte_spider_templates_project"
 
 SPIDER_MODULES = [
-    "zyte_spider_templates.spiders",
```

Comment:
I wonder if we can really do this, thinking about the recent https://github.com/scrapy/scrapy/issues/6544

Reply:
Fixed!

Reply:
You have removed the entire setting. My point is that we probably need to keep it as it is, no changes.

Reply:
Do you mean keep
```
SPIDER_MODULES = [
    "zyte_spider_templates.spiders"
]?
```
I as I got from the https://github.com/scrapy/scrapy/issues/6544 it won't work this way.

Reply:
I mean keep:

```
SPIDER_MODULES = [
    "zyte_spider_templates.spiders",
    "zyte_spider_templates_project.spiders",
]
```

I assume this is the only way you can have both spiders from zyte-spider-templates and custom spiders from your project work both locally (e.g. get listed by `scrapy list`) and in Scrapy Cloud. But better test it out to confirm.

Reply:
Ok, nice, thanks, will return them.